runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -125,66 +160,75 @@ void __quantum__rt__fail_cstr(const char *cstr) { RT_FAIL(cstr); }
 
 void __quantum__rt__initialize()
 {
+    Catalyst::Runtime::RTD_PTR = nullptr;
```

Comment:
I guess there is nothing wrong with resetting the device pointer, but since it seems redundant to me I'm just curious if there was specific reasoning behind it :)